packages/caliper-vscode/extension/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "caliper-vscode-extension",
+	"displayName": "caliper-vscode-extension",
+	"description": "",
+	"version": "0.0.1",
```

Comment:
Every Caliper package version is in sync with each other to give a single "global" version for Caliper. So this "package" should also match the current Caliper version, which is `0.5.1-unstable`